modules/cnf-verify-sriov-operator-n3000.adoc

```diff
@@ -0,0 +1,242 @@
+DPDK_VERSION=20.11
+PCIDEVICE_INTEL_COM_INTEL_FEC_5G=0.0.0.0:1d.00.0 <1>
+~/usr/bin/env
+HOSTNAME=fec-pod
+
+----
++
+<1> This is the PCI address of the virtual function.
++
+[NOTE]
+====
+Depending on the resource that the user requests in the `pod-test.yaml` file, this can be any one of following three PCI addresses:
+
+* PCIDEVICE_INTEL_COM_INTEL_FEC_ACC100
+* PCIDEVICE_INTEL_COM_INTEL_FEC_5G
+* PCIDEVICE_INTEL_COM_INTEL_FEC_LTE
+====
+
+
+.  Change to the `test-bbdev` directory.
++
+[source,terminal]
+----
+sh-4.4# cd test/test-bbdev/
+----
+
+. Check the CPUs that are assigned to the pod:
++
+[source,terminal]
+----
+sh-4.4# export CPU=$(cat /sys/fs/cgroup/cpuset/cpuset.cpus)
+sh-4.4# echo ${CPU}
+----
+This prints out the CPUs that are assigned to the `fec.pod`.
++
+.Example output
+[source,terminal]
+----
+24,25,64,65
+----
+
+. Run the `test-bbdev` application to test the device:
++
+[source,terminal]
+----
+sh-4.4# ./test-bbdev.py -e="-l ${CPU} -a ${PCIDEVICE_INTEL_COM_INTEL_FEC_5G}" -c validation \ -n 64 -b 32 -l 1 -v ./test_vectors/*"
+----
++
+.Example output
+[source,terminal]
+----
+Executing: ../../build/app/dpdk-test-bbdev -l 24-25,64-65 0000:1d.00.0 -- -n 64 -l 1 -c validation -v ./test_vectors/bbdev_null.data -b 32
+EAL: Detected 80 lcore(s)
+EAL: Detected 2 NUMA nodes
+Option -w, --pci-whitelist is deprecated, use -a, --allow option instead
+EAL: Multi-process socket /var/run/dpdk/rte/mp_socket
+EAL: Selected IOVA mode 'VA'
+EAL: Probing VFIO support...
+EAL: VFIO support initialized
+EAL:   using IOMMU type 1 (Type 1)
+EAL: Probe PCI driver: intel_fpga_5ngr_fec_vf (8086:d90) device: 0000:1d.00.0 (socket 1)
+EAL: No legacy callbacks, legacy socket not created
+
+
+
+===========================================================
+Starting Test Suite : BBdev Validation Tests
+Test vector file = ldpc_dec_v7813.data
+Device 0 queue 16 setup failed
+Allocated all queues (id=16) at prio0 on dev0
+Device 0 queue 32 setup failed
+Allocated all queues (id=32) at prio1 on dev0
+Device 0 queue 48 setup failed
+Allocated all queues (id=48) at prio2 on dev0
+Device 0 queue 64 setup failed
+Allocated all queues (id=64) at prio3 on dev0
+Device 0 queue 64 setup failed
+All queues on dev 0 allocated: 64
++ ------------------------------------------------------- +
+== test: validation
+dev:0000:b0:00.0, burst size: 1, num ops: 1, op type: RTE_BBDEV_OP_LDPC_DEC
+Operation latency:
+        avg: 23092 cycles, 10.0838 us
+        min: 23092 cycles, 10.0838 us
+        max: 23092 cycles, 10.0838 us
+TestCase [ 0] : validation_tc passed
+ + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ +
+ + Test Suite Summary : BBdev Validation Tests
+ + Tests Total :        1
+ + Tests Skipped :      0
+ + Tests Passed :       1
+ + Tests Failed :       0
+ + Tests Lasted :       177.67 ms
+ + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ +
+----
++
+[NOTE]
+====
+While some tests can be skipped, be sure that the vector tests pass.
```

Comment:
Unwrap this as a note and move it before the `.Example output` command or put a callout in the example output and use this text in the callout description.